### PR TITLE
Update spec fixtures to concat 2.x

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: .fixtures.yml [hunk 1]
```diff
@@ -5,10 +5,7 @@ fixtures:
     apt:
       repo:           'git://github.com/puppetlabs/puppetlabs-apt'
       branch:         '1.8.x'
-    concat:
-      repo:           'git://github.com/puppetlabs/puppetlabs-concat'
-      branch:         '1.2.x'
-    file_concat:      'git://github.com/electrical/puppet-lib-file_concat'
+    concat:           'git://github.com/puppetlabs/puppetlabs-concat'
```
Comment (ekohl) [inline .fixtures.yml]: I think puppetlabs-apache still needs 1.2.x.

File: .fixtures.yml [hunk 2]
```diff
@@ -5,10 +5,7 @@ fixtures:
     apt:
       repo:           'git://github.com/puppetlabs/puppetlabs-apt'
       branch:         '1.8.x'
-    concat:
-      repo:           'git://github.com/puppetlabs/puppetlabs-concat'
-      branch:         '1.2.x'
-    file_concat:      'git://github.com/electrical/puppet-lib-file_concat'
```
Comment (ekohl) [inline .fixtures.yml]: This is needed with puppetlabs-concat master.

File: .fixtures.yml (hunk 1 above)
Comment (domcleal) [inline .fixtures.yml]: It claims to, but it's actually compatible.  I just opened a PR to relax the dep and fix a couple of minor deprecations.

File: .fixtures.yml (hunk 2 above)
Comment (domcleal) [inline .fixtures.yml]: It was, but was dropped before 2.0.0 shipped during this refactor: https://github.com/puppetlabs/puppetlabs-concat/pull/306

File: .fixtures.yml (hunk 1 above)
Comment (domcleal) [inline .fixtures.yml]: https://github.com/puppetlabs/puppetlabs-apache/pull/1126

File: .fixtures.yml (hunk 2 above)
Comment (ekohl) [inline .fixtures.yml]: Yes, I was looking at the changelog and noticed the same thing.

**Comment by ekohl** [comment]
:+1:

**Comment by mmoll** [comment]
thanks @domcleal!

**Pull request merged**